Cargo.toml

```
@@ -123,22 +125,22 @@ pico-args = { version = "0.5.0", features = [
 png = "0.17.16"
 pollster = "0.4"
 profiling = { version = "1", default-features = false }
-raw-window-handle = "0.6"
+raw-window-handle = { version = "0.6", default-features = false }
 rayon = "1"
 renderdoc-sys = "1.1.0"
 ron = "0.8"
 # NOTE: rustc-hash v2 is a completely different hasher with different performance characteristics
 # see discussion here (including with some other alternatives): https://github.com/gfx-rs/wgpu/issues/6999
 # (using default-features = false to support no-std build, avoiding any extra features that may require std::collections)
 rustc-hash = { version = "1", default-features = false }
-serde_json = "1.0.137"
+serde_json = { version = "1.0.137", default-features = false }
```

I would love to update the version as well:

```suggestion
serde_json = { version = "1.0.138", default-features = false }
```

but this would inflate the changes with the updates to `Cargo.lock`. I will likely leave this version update for another PR (someday).